libraries/botbuilder-dialogs-declarative/tests/schemaMergeTest.js

```diff
@@ -3,32 +3,21 @@ const path = require('path');
 const assert = require('assert');
 const fs = require('fs');
 const Validator = require('jsonschema').Validator;
+const util = require('util');
 // eslint-disable-next-line security/detect-child-process
-const { exec } = require('child_process');
+const exec = util.promisify(require('child_process').exec);
 
 // Note: This file is intentionally not named *.test.js to ensure it isn't run
 // via `yarn run test` or `npm run test`.
 
-const maybeIt = process.env.FORCE_SCHEMA_MERGE || process.platform === 'win32' ? it : it.skip;
-
 async function runCommand(command) {
-    return new Promise((resolve, reject) => {
-        exec(
-            command,
-            {
-                cwd: __dirname,
-            },
-            (err, stdout, stderr) => {
-                // stderr may be used by VS Code Debugger, even without errors, so we ignore it in DEBUG environment.
-                // Note: You have to manually set "env": { "DEBUG": "true" } in launch.json for this check to work.
-                if (err || (stderr && process.env.DEBUG !== 'true')) {
-                    reject(err || stderr);
-                } else {
-                    resolve(stdout);
-                }
-            }
-        );
-    });
+    const { stdout, stderr } = await exec(command);
+
+    if (stderr) {
+        throw stderr;
+    }
+
+    return { stdout, stderr };
```

Comment:
Since you're throwing every time `stderr` is truthy, you may as well just `return stdout` here.